src/components/DocumentationTopic.vue

```diff
@@ -647,6 +676,47 @@ export default {
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
 
+$space-size: 15px;
+
+.declaration-list-menu {
+  position: relative;
+  margin: 0 !important;
+  width: 100%;
+
+  .declaration-list-toggle {
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background: var(--color-fill); // cover border line
+    padding: 5px 15px;
+    color: var(--colors-link, var(--color-tutorials-overview-link));
```

Comment:
Is there a reason why we can't just use `var(--color-link)` here? It seems a little odd to reference a tutorial-specific color here.